rotkehlchen/db/db_cache.py

```diff
@@ -0,0 +1,74 @@
+from enum import auto
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.types import Timestamp
+from rotkehlchen.utils.mixins.enums import Enum
```

Comment:
change the name of this file. It's already under the DB directory. Name it `cache.py`